### DISCUSSION TITLE
Radeon RX 5700 XT

### DISCUSSION THREAD

**Discussion by starlisa** [lllyasviel/Fooocus#2939] [Q&A]
When I try installing Fooocus, I get a runtime error saying that no NVIDIA driver was found. My graphics card isn't NVIDIA, though. It's a Radeon RX 5700 XT. Can I install an NVIDIA driver with a Radeon GPU? Note: I'm using Windows 11. I've tried the change listed on https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#windows-amd-gpus but it continues to complain that I don't have an NVIDIA driver.

**Comment by mashb1t** [comment]
Please make sure to follow all steps mentioned in the readme. Make sure you've manually installed all necessary packages and you're not by accident using a local python version instead of the embedded (when using the zip file installation method) or the one in your venv (manual setup method).

zip file method:
```
.\python_embeded\python.exe -m pip uninstall torch torchvision torchaudio torchtext functorch xformers -y
.\python_embeded\python.exe -m pip install torch-directml
.\python_embeded\python.exe -s Fooocus\entry_with_update.py --directml
pause
```

Please provide your complete terminal output for further debugging either here or in a new issue using our issue template.

**Comment by hqnicolas** [comment]
https://github.com/lllyasviel/Fooocus/discussions/2552
https://github.com/hqnicolas/lllyasvielFooocusROCm/tree/main
